h/static/scripts/group-forms/components/CreateEditGroupForm.tsx

```diff
@@ -290,10 +296,25 @@ export default function CreateEditGroupForm({
                 <GlobeIcon /> Open
               </RadioGroup.Radio>
             </RadioGroup>
-          </>
+          </div>
+        )}
+
+        {config.features.group_moderation && (
+          <div>
+            <Label text="Moderation" />
```

Comment:
This isn't semantically a label for a single form control but rather a heading which we want to style in a way that matches labels above. More appropriate would be say `<fieldset>` + `<legend>` or `<section>` + `<h2>`.

Reply:
Oh, right. I actually wanted to revisit this but I ended up forgetting.

Reply:
I replaced the containing `div` with a `fieldset`, and added a `legend` with the same content and font weight. Visually everything looks the same.